oss-internship-2020/curl/sandbox.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <linux/futex.h>
+#include <sys/mman.h>  // For mmap arguments
+#include <syscall.h>
+
+#include <cstdlib>
+
+#include "curl_sapi.sapi.h"
+#include "sandboxed_api/sandbox2/util/bpf_helper.h"
+#include "sandboxed_api/util/flag.h"
+
+class CurlSapiSandbox : public CurlSandbox {
+ protected:
```

Comment:
This can even be `private`

Reply:
It seems like it can't be made `private` because the method is used by `CurlSapiSandboxEx3`, a subclass of `CurlSapiSandbox` defined in `examples/example3.cc`.